packages/suite-data/files/translations/en.json

```
@@ -45,6 +45,7 @@
   "REVIEW_AND_SEND_TRANSACTION": "Review & Send",
   "SETTINGS_ADV_COIN_BLOCKFROST_DESCRIPTION": "Trezor Suite uses Blockfrost websocket-link for the wallet backend.",
   "SETTINGS_ADV_COIN_CONN_INFO_NO_CONNECTED": "Not connected to the backend. Try reconnecting the device. Also, check your internet connection or the URL of your custom backend.",
+  "SHAMIR_SEED": "Advanced shamir share backup",
```

```suggestion
  "SHAMIR_SEED": "Advanced Shamir share backup",
```

Or `Advanced Shamir Backup`